charts/gha-runner-scale-set/templates/_helpers.tpl

```diff
@@ -93,19 +93,26 @@ volumeMounts:
 
 {{- define "gha-runner-scale-set.dind-container" -}}
 image: docker:dind
+args:
+  - dockerd
+  - --host=unix:///run/docker/docker.sock
+  - --group=$(DOCKER_GROUP_GID)
+env:
+  - name: DOCKER_GROUP_GID
+    value: "123"
```

Comment:
This is could potentially cause issues especially since we don't provide a way to configure it. It's true we set `123` in our runner image even: https://github.com/actions/runner/blob/2908d82845c018193655e68f3c20a5ad03bc0efd/images/Dockerfile#L43 but the idea from the runner image is that users can override and build an image that works best for them.

I'm not sure how to address this without exposing the `dind` configuration through the helm charts which we intentionally left out.

Reply:
@Link- Do you have any ideas how to better deal with this?

Reply:
I'll keep you posted

Reply:
I don't see an alternative. Let's move forward with it.